### PR TITLE
/config: Added `Copy to clipboard` button for /config

### PR DESCRIPTION
**Pull request by staff0rd**
Allows easy copying of `/config` to clipboard via button click because drag-selecting is boring.

/cc @juliusv

### CODE REVIEW

**Comment by simonpasquier** [comment]
Would you be able to resolve the conflicts? It seems like a useful addition to me.

**Comment by staff0rd** [comment]
@simonpasquier done

image

File: web/ui/static/css/config.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.btn {
+    border-radius: 0;
+}
```
Comment (juliusv) [inline web/ui/static/css/config.css]: Please end text files with a newline (same below).

Comment (staff0rd) [reply web/ui/static/css/config.css]: resolved in 9349188

Comment (juliusv) [reply web/ui/static/css/config.css]: Hmm, now the files end with two newlines. I just meant not removing the trailing newline that should be there, as is standard in Unix/POSIX text files, see: https://stackoverflow.com/a/729795

All common editors will have a configuration setting to not remove the final newline character upon saving.

**Comment by juliusv** [comment]
👍 thanks!

**Pull request merged**